r/rasoa-kininike.md

```diff
@@ -0,0 +1,9 @@
+# Rasoa Kininike
+
+![Rasoa Kininike](rasoa-kininike.JPG)
+
+This artist is born in 1978 and she was died in 2014 by bus accident. She is from Tongobory-Toliara.She is Lico Kininike's mother.
```

Comment:
`This artist was born in 1978 and died in 2014 by bus accident. She is from Tongobory-Toliara. She is Lico Kininike's mother.`